### PR TITLE
Add support for handling multiple arguments

### PR DESCRIPTION
**Pull request by Mitch528**
This PR adds support for subscribing to events that pass in multiple arguments. It also fixes handles not being removed on disposal.

### CODE REVIEW

**Comment by danroth27** [comment]
@galvesribeiro Any update on this PR? I was going to use this library for a SignalR demo, but I needed support for multiple args as well.

**Comment by galvesribeiro** [comment]
Sorry folks. I'll look at it later today as soon as I have time and will release the package. Sorry about the delay, busy days.

File: src/Blazor.Extensions.SignalR/HubConnection.cs [hunk 1]
```diff
@@ -37,21 +37,120 @@ public HubConnection(HttpConnectionOptions options)
...
+            Func<TResult1, TResult2, TResult3, TResult4, TResult5, TResult6, TResult7, TResult8, Task> handler)
+            => On<TResult1, TResult2, TResult3, TResult4, TResult5, TResult6, TResult7, TResult8, object, object>(methodName,
+                (t1, t2, t3, t4, t5, t6, t7, t8, t9, t10) => handler(t1, t2, t3, t4, t5, t6, t7, t8));
+
+        public IDisposable On<TResult1, TResult2, TResult3, TResult4, TResult5, TResult6, TResult7, TResult8, TResult9>(string methodName,
+            Func<TResult1, TResult2, TResult3, TResult4, TResult5, TResult6, TResult7, TResult8, TResult9, Task> handler)
+            => On<TResult1, TResult2, TResult3, TResult4, TResult5, TResult6, TResult7, TResult8, TResult9, object>(methodName,
+                (t1, t2, t3, t4, t5, t6, t7, t8, t9, t10) => handler(t1, t2, t3, t4, t5, t6, t7, t8, t9));
+
+        public IDisposable On<TResult1, TResult2, TResult3, TResult4, TResult5, TResult6, TResult7, TResult8, TResult9, TResult10>(string methodName,
```
Comment (galvesribeiro) [inline src/Blazor.Extensions.SignalR/HubConnection.cs]: I was never a fan of those multiple `TResultXXX` parameters/overloads but that is how .Net deal with generic overloads so there is nothing we can do. Besides, the original C# client does that as well, so why not? :)

File: src/Blazor.Extensions.SignalR/HubConnection.cs [hunk 2]
```diff
@@ -37,21 +37,120 @@ public HubConnection(HttpConnectionOptions options)
...
+                     return handler(t1, t2, t3, t4, t5, t6, t7, t8, t9, t10);
                  }
             );
 
+            RegisterHandle(methodName, callback);
+
+            return callback;
+        }
+
+        internal void RegisterHandle(string methodName, HubMethodCallback callback)
```
Comment (galvesribeiro) [inline src/Blazor.Extensions.SignalR/HubConnection.cs]: NIT: That could become an inline function as it is only used on the previous method. Will look at that later on.

**Pull request merged**